City-of-Pori/pori-kada
# PORI-62 Small lifup banner (also known as Rauma banner)

To test:

1. drush fra -y && drush cc all
2. Go to edit any liftup and choose from liftup options "to small lifup banner". Also make sure the liftup is published.
3. Add four liftups this way and make sure some of them have internal links, external links and some without any links.
4. Go to front page and make sure there is a new banner just before footer that has these liftups you just added there.
5. Make sure the banner works responsively.
